checkov/arm/checks/resource/ACRGeoreplicated.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+from typing import Any
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.arm.base_resource_check import BaseResourceCheck
+
+
+class ACRGeoreplicated(BaseResourceCheck):
+    def __init__(self) -> None:
+
+        name = "Ensure geo-replicated container registries to match multi-region container deployments."
+        id = "CKV_AZURE_165"
+        supported_resources = ("Microsoft.ContainerRegistry/registries/replications",)
+        categories = (CheckCategories.NETWORKING,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf: dict[str, list[Any]]) -> CheckResult:
+        sku = conf.get("sku")
```

Comment:
i don't see SKU value here - 
https://learn.microsoft.com/en-us/azure/templates/microsoft.containerregistry/registries/replications?pivots=deployment-language-arm-template

Reply:
Please check it